crates/jstzd/src/task/file.rs

```diff
@@ -0,0 +1,104 @@
+use std::fs::File;
+use std::path::PathBuf;
+
+use anyhow::Result;
+use tempfile::NamedTempFile;
+
+#[derive(Debug)]
+pub enum FileWrapper {
```

Comment:
nit we could just name it as `File` similar to `Directory`?

Reply:
There are already too many `File`s in this module, so I decided to add `Wrapper` to avoid confusion.